crates/blockifier/src/concurrency/worker_logic_test.rs

```diff
@@ -105,7 +106,7 @@ pub fn test_commit_tx() {
     let cached_state =
         test_state(&block_context.chain_info, BALANCE, &[(account, 1), (test_contract, 1)]);
     let versioned_state = safe_versioned_state_for_testing(cached_state);
-    let executor =
+    let executor: WorkerExecutor<'_, _, VisitedPcsSet> =
```

Comment:
Necessary type annotation? If it is, it occurs a number of times here and below, so please make it an alias if possible.

Reply:
`WorkerExecutor` requires the type annotation because it can't be inferred from the arguments of the constructor.